docker-compose.yml

```
@@ -58,7 +58,7 @@ services:
   keycloak-config-cli:
     profiles:
       - config
-    image: adorsys/keycloak-config-cli:latest
+    image: adorsys/keycloak-config-cli:latest-11.0.3
```

```suggestion
    image: adorsys/keycloak-config-cli:latest-${KEYCLOAK_VERSION}
```
But I've done that one in snapshots+ anyway :)

---

🤔 it's a little more complicated. We use `KEYCLOAK_VERSION = 11.0.0` (it's the version in production), but this tool is not working with that version, needs the fixed client in version `11.0.3`.

We need to be able to override the `KEYCLOAK_CONFIG_CLI_VERSION`, right now we are hardcoded into `11.0.0` and `11.0.3`.

---

couldn't we use the patched version in production ? It seems sensible anyway :)

---

I wish, I don't know how infra is selecting the versions. Is not easy/quick to perform those upgrades for them (they need to plan the upgrade) and they perform all the steps manually.